sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -1926,9 +1926,21 @@ void CAFMaker::produce(art::Event& evt) noexcept {
 
     // NOTE: The sbn::crt::CRTHit is associated to the T0. It's a bit awkward to
     // access that here, so we do it per-track (see code where fmCRTHitMatch is accessed below)
+   
+    // note: in the ICARUS case we do not need a slice_tag_stuff for CRTHitMatch as it is
+    // common to East and West
+    /*
     art::FindManyP<anab::T0> fmCRTHitMatch =
       FindManyPStrict<anab::T0>(slcTracks, evt,
                fParams.CRTHitMatchLabel() + slice_tag_suff);
```

Comment:
I'm hesitant to just get rid off the `slice_tag_suff`, as I suspect SBND may need it. Summoning @henrylay97 to comment on this

Reply:
Could anybody explain how SBND stores the tracks? ICARUS splits them in two data products by their cryostat, which I assume SBND does not.

Reply:
At SBND we store our tracks as a single collection - due to the fact we have just one cryostat. So we don't use the suffix approach.